src/ViewModel/Factory/SiteHeaderFactory.php

```diff
@@ -156,7 +156,7 @@ public function createSiteHeader(Model $item = null, Profile $profile = null) :
         } else {
             $secondaryLinks[] = NavLinkedItem::asLoginControl(
                 LoginControl::notLoggedIn(
-                    mixed_visibility_text('', 'Log in/Register', '(via ORCID)'),
+                    mixed_visibility_text('', 'Log in/Register', '(via ORCID - An ORCID is a persistent digital identifier for researchers)'),
```

Comment:
Seems to be _more_ branding rather than something useful?

Reply:
@thewilkybarkid This was what ORCID wanted on the Author information section but having talked to @davidcmoulton we made the compromise to put it in the login/register button.

@davidcmoulton Do we need to revise this again?

Reply:
I think this provides useful spoken context to a user who may not have heard of ORCID. Happy to proceed.